compiler/passes/initializerRules.cpp

```diff
@@ -373,6 +373,51 @@ static void preNormalizeInitClass(FnSymbol* fn) {
   }
 }
 
+static void checkLocalPhaseOneErrors(InitNormalize state,
```

Comment:
I think this *could* be "const InitNormalize& state" but I am not worried about it.